app/models/ancestor_ref.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class AncestorRef < ActiveRecord::Base
+  DEFAULT_ANCESTOR_BRANCH = 'master'
+
+  fields do
+    ref          :string, limit: 40,  required: true, default: 'master'
```

Comment:
Any specific reason for a 40 char limit?

Reply:
that is the string length of a git sha.. I updated it to 255 characters to be more flexible. The fact that it is a `ref` means it could actually be set to a branch name in the future, in which case the 40 char limit doesn't make sense